configs/components/jsbach/jsbach.yaml

```diff
@@ -289,7 +289,8 @@ forcing_in_work:
         "a_nd_file": "a_nd_file.nc"
         "lightning": "lightning.nc"
         "popdens": "population_density.nc"
-        "LU" : "landuse_harvest.@YEAR@.nc"
+        # Naming based upon: https://gitlab.awi.de/paleodyn/Models/echam6/-/blob/6.3.05p2-awiesm-2.1/src/src_jsbach3/mo_cbal_landcover_change.f90#L3050 
+        "LU" : "landuseHarvest.@YEAR@.nc"
```

Comment:
To be cleaned, maybe: we have `LU_trans` below, maybe we should change here to `LU_harvest` here. In principle, I would also be happy with changing some of the other left-hand-side names to be more human understandable, e.g. `population_density` instead of `popdens`, and `a_nd_file`...whatever that is? It will help people also understand what JSBACH is doing.